frontend/src/core/api/base.js

```diff
@@ -88,7 +88,7 @@ export default class APIBase {
         }
     }
 
-    keysToCamelCase(results) {
+    keysToCamelCase(results: Object) {
```

Comment:
`Object` is an alias of `any` and it is recommended by Flow not to use it. You can instead just use `any` here, as we don't really care what the type of value is.